python/ray/_private/worker.py

```diff
@@ -2927,6 +2984,7 @@ def put(
     value: Any,
     *,
     _owner: Optional["ray.actor.ActorHandle"] = None,
+    tensor_transport: str = "object_store",
```

Comment:
Sorry forgot to mention this earlier - let's also make this `_tensor_transport` since it is still an experimental API. You can also add **alpha** to the docstring.